analyzer/requirements_py/dev/requirements.txt

```diff
@@ -1,3 +1,4 @@
+setuptools-rust==0.12.1
```

Comment:
This is not the best solution because if we want to build only the web part and create a virtual environment in the web directory it will still fail. If we move this depdency to the web requirement file it will still fail because it is need during the installation phase.

The original problem comes from the `cryptography` package which is needed by the `twine` package and which is installed in this file: https://github.com/Ericsson/codechecker/blob/344668101fd483b5631961c6df988b05bf536590/web/requirements_py/dev/requirements.txt#L19

The behaviour is changed in the `cryptography 3.4` version. So if we use an older version from it the problem will be dissapeared. So I recommend the following:

Modify the line here: https://github.com/Ericsson/codechecker/blob/344668101fd483b5631961c6df988b05bf536590/web/requirements_py/dev/requirements.txt#L19
And insert the following line above it: `cryptography==3.3.2`
This way no need to install `rustc` compiler and this `setuptools-rust` package.

What do you think?

Reply:
I don't think it is a good idea to get stuch with an old cryptography==3.3.2 dependency.

Wouldn't it be a solution to add it to the codechecker/web/requirements_py/dev/requirements.txt  too?

Reply:
I'm not sure which step requires this "rust" thing but it can be turned off by providing `CRYPTOGRAPHY_DONT_BUILD_RUST=1` before that step.